packages/presets/src/fragments/frame-panel/card/frame-card.ts

```diff
@@ -211,7 +211,7 @@ export class FrameCard extends WithDisposable(LitElement) {
     });
   }
 
-  private _renderDraggingCardNumber() {
+  private DraggingCardNumber() {
```

Comment:
Better use `_DraggingCardNumber` for private templates 🌝

Reply:
Get it.